model/src/main/proto/topic.proto

```diff
@@ -671,3 +671,23 @@ message SubtopicRecord {
   // The old title corresponding to the subtopic (as a string).
   reserved 1;
 }
+
+// Top-level proto used to store the aggregate learning duration timestamps in a topic,
+// per-profile basis.
+message TopicLearningTimeDatabase {
+  // Map from topic ID to TopicLearningTime.
+  map<string, TopicLearningTime> aggregate_topic_learning_time = 1;
+}
+
+// Represents the active learning time spent in a topic.
+message TopicLearningTime {
+  // The ID of the topic for which this learning time has been recorded.
+  string topic_id = 1;
+
+  // The total time spent by a learner in this topic.
+  int64 topic_learning_time_ms = 2;
+
+  // The last time the topic_aggregate_learning_time_ms was updated, coinciding with the last time
+  // the learner played an exploration in this topic.
```

Comment:
I suggest indicating that this is a Unix epoch timestamp in milliseconds (as opposed to a duration or any other type of time value that could be stored here).

Reply:
I updated this comment